### DISCUSSION TITLE
Automate Update Row

### DISCUSSION THREAD

**Discussion by olegrand1976** [Budibase/budibase#6926] [Help]
Hello,

   I have been using BUDIBASE for 1 week and I think it is great! 

I am currently a stuck because I would like to update the value of a field in a table via an automated action.

I have 2 tables CONTRACTS and PRODUCER.

I would like to update the CONTRACTS's COMMISSION_NET field with a calculation : 

CONTRACTS.COMMISSION_NET = CONTRACTS.PREMIUM_OUT_OF_TAX * CONTRACTS.PRODUCER.RATIO

Here is the information from my tables:

[CONTRACT.csv] (https://github.com/Budibase/budibase/files/9200182/CONTRAT.csv)
[PRODUCER.csv] (https://github.com/Budibase/budibase/files/9200183/PRODUCTEUR.csv)

How do I do this?

I have tried a few times but I can't get it to work.

**Comment by melohagan** [comment]
Hey @olegrand1976 

Thanks for trying out Budibase!

I'm assuming you have established a relationship between the two tables within Budibase? https://docs.budibase.com/docs/relationships

Is there any reason why you want or need to use automations for this? You could create a formula column in your CONTRACTS table: https://docs.budibase.com/docs/formula
